.ci/packaging/request_to_obs.sh

```diff
@@ -55,13 +55,25 @@ echo "INFO: BUILD_HEAD=${BUILD_HEAD}"
 echo "INFO: BRANCH=${BRANCH}"
 echo "INFO: OBS_BRANCH=${OBS_SUBPROJECT}"
 echo "INFO: PUSH=${PUSH}"
+echo "INFO: SUBPROJECT_TYPE=${SUBPROJECT_TYPE}"
 
 # Export in all pipeline tasks
 cd "${packaging_dir}/obs-packaging"  || exit 1
+gen_versions_cmd="./gen_versions_txt.sh"
+if [ BUILD_HEAD = "true" ];then
 echo "Building for head gen versions ..."
-./gen_versions_txt.sh --head "${BRANCH}"
+gen_versions_cmd=" --head"
```

Comment:
This looks wrong as you are assigning the variable. Shouldn't you be appending?:

```sh
gen_versions_cmd+=" --head"
```